server/src/database/models/patient-general-therapy/schema.sql

```diff
@@ -0,0 +1,10 @@
+DROP TABLE IF EXISTS "patient_general_therapy" CASCADE;
+
+CREATE TABLE "patient_general_therapy" (
+  "id" SERIAL PRIMARY KEY,
+  "patient_id" INTEGER REFERENCES patients(id) NOT NULL,
+  "global_therapy_id" INTEGER REFERENCES global_therapy(id) NOT NULL,
```

Comment:
```suggestion
  "general_therapy_id" INTEGER REFERENCES general_therapy(id) NOT NULL,
```
Looks like you forgot to edit the name here.

Reply:
done